bin/auditwheel

```diff
@@ -22,23 +22,14 @@ libs = [
     "libcusparse.so.11",
     "libcusparse.so.12",
     "libcustatevec.so",
+    "libcustatevec.so.0",
     "libcustatevec.so.1",
```

Comment:
Do we need all three?

Reply:
Yes. There's no guarantee on the naming, as it can differ between the flavour of the Cuda toolkits.